NEWS.md

```diff
@@ -151,32 +151,25 @@ Library improvements
-    messages, simply set `ENV["JULIA_WARN_COLOR"] = 208`.  Please note that not
-    all terminals support 256 colors.
-
-  * The function `print_with_color` no longer prints text in bold by default
-    ([#18628]).  Instead, the function now take a keyword argument `bold::Bool`
-    which determines whether to print in bold or not.  On some terminals,
-    printing a color in non bold results in slightly darker colors being
-    printed than when printing in bold.  Therefore, light versions of the
-    colors are now supported.  For the available colors see the help entry on
-    `print_with_color`.
-
-  * The default color for info messages has been changed from blue to cyan and
-    for warning messages from red to yellow.  This can be changed back to the
-    original colors by setting the environment variables `JULIA_INFO_COLOR` to
-    `"blue"` and `JULIA_WARN_COLOR` to `"red"`.  One way of doing this is by
-    adding for example `ENV["JULIA_INFO_COLOR"] = :blue` and
-    `ENV["JULIA_WARN_COLOR"] = :red` to the `.juliarc.jl` file.  For more
-    information regarding customizing colors in the REPL, see this [manual
-    section](
-    http://docs.julialang.org/en/latest/manual/interacting-with-julia/#customizing-colors).
+  * The function `print_with_color` can now take a color represented by an integer between 0 and 255 inclusive as its first argument ([#18473]).
+    For a number to color mapping please refer to [this chart](https://upload.wikimedia.org/wikipedia/en/1/15/Xterm_256color_chart.svg).
+    It is also possible to use numbers as colors in environment variables that customizes colors in the REPL.
+    For example, to get orange warning messages, simply set `ENV["JULIA_WARN_COLOR"] = 208`.
+    Please note that not all terminals support 256 colors.
+
+  * The function `print_with_color` no longer prints text in bold by default ([#18628]).
+    Instead, the function now take a keyword argument `bold::Bool` which determines whether to print in bold or not.
+    On some terminals, printing a color in non bold results in slightly darker colors being printed than when printing in bold.
+    Therefore, light versions of the colors are now supported.
+    For the available colors see the help entry on `print_with_color`.
+
+  * The default text style for REPL input and answers has been changed from bold to normal ([#11250]).
+    They can be changed back to bold by setting the environment variables `JULIA_INPUT_COLOR` and `JULIA_ANSWER_COLOR` to `"bold"`.
+    For example, one way of doing this is adding `ENV["JULIA_INPUT_COLOR"] = :bold` and `ENV["JULIA_ANSWER_COLOR"] = :bold` to the `.juliarc.jl` file.
+    See the [manual section on customizing colors](http://docs.julialang.org/en/latest/manual/interacting-with-julia/#customizing-colors) for more information.
+
+  * The default color for info messages has been changed from blue to cyan, and for warning messages from red to yellow.
```

Comment:
this should probably have its issue/pr reference added, it got lost at some point